.gitignore

```diff
@@ -136,3 +136,4 @@ doc/source/dev/schema.rst
 *.rej
 *~
 .idea/
+.vscode/
```

Comment:
Definitely minor, but I think things like this (visual studio, intellij, etc. project caches) should be put in the individual user's global .gitignore instead of here, where it's irrelevant to most other contributors.